packages/cf-style-container/src/createComponent.js

```diff
@@ -0,0 +1,72 @@
+import { extractPassThroughProps, resolvePassThrough } from 'fela-utils';
+import { createElement } from 'react';
+import PropTypes from 'prop-types';
+
+function createComponentFactory(createElement, contextTypes) {
+  return function createComponent(rule, type, passThroughProps) {
+    const displayName = rule.name ? rule.name : 'FelaComponent';
+
+    const FelaComponent = ({ children, ...ruleProps }, { renderer, theme }) => {
+      if (!renderer) {
+        throw new Error(
+          "createComponent() can't render styles without the renderer in the context. Missing react-fela's <Provider /> at the app root?"
```

Comment:
But won't that then crash like in [this bug report](https://github.com/cloudflare/cf-ui/issues/234#issuecomment-310485817) since it won't have a base theme?